app/uk/gov/hmrc/cataloguefrontend/servicecommissioningstatus/ServiceCommissioningStatusController.scala

```diff
@@ -89,18 +89,33 @@ class ServiceCommissioningStatusController @Inject() (
                                 allChecks <- serviceCommissioningStatusConnector.allChecks()
                                 results   <- serviceCommissioningStatusConnector.cachedCommissioningStatus(formObject.teamName, formObject.serviceType)
                                 checks    =  if (formObject.checks.isEmpty) allChecks.map(_._1).toList else formObject.checks
-
+                                
+                                filteredResults = results.filter { result => //exclude decommissioned services
```

Comment:
It might be clearer to amend the comment to "we show all service with any present checks since this can indicate where they haven't been fully decommissioned"

Reply:
👍